homeassistant/components/smartthings/__init__.py

```diff
@@ -132,11 +131,17 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             entry.data[CONF_REFRESH_TOKEN],
         )
         hass.config_entries.async_update_entry(
-            entry, data={**entry.data, CONF_REFRESH_TOKEN: token.refresh_token}
+            entry,
+            data={
+                **entry.data,
+                CONF_ACCESS_TOKEN: token.access_token,
+                CONF_REFRESH_TOKEN: token.refresh_token,
+            },
         )
 
         # Get devices and their current status
-        devices = await api.devices(location_ids=[installed_app.location_id])
+        devices = await api.devices()
+        print(devices)
```

Comment:
Remove all print statements

Reply:
It's also not ready for review. This is still a debugging thing to share progress